docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -55,6 +55,14 @@ We have added 4 new features to the flamegraph visualization:
 
 {{< video-embed src="/media/docs/grafana/panels-visualizations/screen-recording-grafana-10.1-flamegraph-whatsnew.mp4" >}}
 
+### Visualizations and Widgets split
+
+<!-- Alexa Vargas -->
+
+_Experimental in some of our free tiers of Grafana._
```

Comment:
Is this how we communicate it? I'd expect to follow the same as any other release type "_Experimental in all editions of Grafana._" In the end it's available for everyone but they'd have to enable the feature toggle.

Reply:
I'm wondering the same here. As well as whether we should add something about feature toggles.